sdcm/cluster.py

```diff
@@ -2836,6 +2836,19 @@ def get_nodetool_status(self, verification_node=None):  # pylint: disable=too-ma
                         pass
         return status
 
+    @staticmethod
+    def get_nodetool_info(node):
+        """
+            Runs nodetool info and generates status structure.
+            Info format:
+
+            :param node: node to run the nodetool on
+            :return: dict
+        """
+        res = node.run_nodetool('info')
+        info_res = yaml.load(res.stdout)
+        return info_res
```

Comment:
why json? 
```
nodetool info
ID                     : 3722bbba-5751-4900-8c45-4e02483b816d
Gossip active          : true
Thrift active          : true
Native Transport active: true
Load                   : 539.38 KB
Generation No          : 1576121406
Uptime (seconds)       : 1735
Heap Memory (MB)       : 7.97 / 247.50
Off Heap Memory (MB)   : 66.15
Data Center            : us-east
Rack                   : 1b
Exceptions             : 0
Key Cache              : entries 0, size 0 bytes, capacity 0 bytes, 0 hits, 0 requests, 0.000 recent hit rate, 0 save period in seconds
Row Cache              : entries 130, size 130 bytes, capacity 11.48 MiB, 6646 hits, 6646 requests, 1.000 recent hit rate, 0 save period in seconds
Counter Cache          : entries 0, size 0 bytes, capacity 0 bytes, 0 hits, 0 requests, 0.000 recent hit rate, 0 save period in seconds
Percent Repaired       : 0.0%
Token                  : (invoke with -T/--tokens to see all 256 tokens)

```

Reply:
good point, I guess it wasn't yet used any of the data there. even that it would be very interesting to see if yaml can parse this thing directly, 
and if it does, this is a good usage of it :)

Reply:
and this actually works just nicely. i'm also surprised @fgelcer you get  :1st_place_medal:

Reply:
yes, yaml, not json. Great that it works

Reply:
this is the easiest way to convert this kind of content to a python dict(), and back, and it can also read and write stream straight to a file (like a config file).